package.json

```diff
@@ -0,0 +1,49 @@
+{
+	"name": "blank-theme",
+	"version": "2.0.0",
+	"description": "Blank Theme Packages",
+	"author": "rtCamp",
+	"license": "MIT",
+	"private": true,
+	"scripts": {
+		"prod": "NODE_ENV=production webpack --progress",
+		"dev": "webpack --watch --progress",
+		"clean": "rm -rf assets/build/*",
+		"init": "./bin/init.js && npm run pot && rm -rf .git .github ./bin/ ./languages/blank-theme.pot && npm uninstall replace-in-file prompt-sync",
+		"lint:css": "stylelint assets/src/sass/**/*.scss --syntax scss",
+		"lint:js": "eslint assets/src/js/",
+		"pot": "wp-pot --src './**/*.php' --dest-file './languages/blank-theme.pot' --domain 'blank-theme' --package 'Blank Theme' ",
+		"precommit": "npm run lint-css && npm run lint-js && npm run pot"
+	},
+	"dependencies": {
+		"foundation-sites": "^6.6.3",
+		"webfontloader": "^1.6.28"
+	},
+	"devDependencies": {
+		"@wordpress/babel-preset-default": "^6.2.0",
+		"@wordpress/browserslist-config": "^4.0.1",
+		"@wordpress/dependency-extraction-webpack-plugin": "^3.1.4",
+		"@wordpress/eslint-plugin": "^9.0.6",
+		"@wordpress/stylelint-config": "^19.0.5",
+		"babel-loader": "^8.2.2",
+		"css-loader": "^5.2.6",
+		"eslint": "^7.27.0",
+		"eslint-webpack-plugin": "^2.5.4",
+		"mini-css-extract-plugin": "^1.6.0",
+		"postcss": "^8.3.0",
+		"postcss-loader": "^5.3.0",
+		"postcss-preset-env": "^6.7.0",
+		"prompt-sync": "^4.2.0",
+		"replace-in-file": "^6.2.0",
+		"sass": "^1.34.0",
+		"sass-loader": "^12.0.0",
+		"stylelint": "^13.13.1",
+		"stylelint-webpack-plugin": "^2.1.1",
+		"webpack": "^5.38.1",
+		"webpack-cli": "^4.7.0",
+		"wp-pot-cli": "^1.5.0"
```

Comment:
We can also include https://www.npmjs.com/package/postcss-sort-media-queries to sort and combine media queries. This will reduce the CSS file size as well.

Or let me know if any other packages do the same job.

Reply:
I've removed it for only reason that it can have some side effects that we have not been keeping eye on. I would like to revisit it maybe once I use this new changes on one of the client project with properly defined breakpoints.